src/test/input.ts

```diff
@@ -0,0 +1,42 @@
+export const user1 = `#graphql
+mutation {
+    createUser (data: {
+      name: "eu",
+      email: "eu@gmail.com",
+      birthDate: "27/12/1900",
+      password: "mudar123"
+    } 
+  ) {
+      birthDate
+      email
+      id
+      name
+  }
+}`;
+
+export const queryUser = `#graphql
+query {
+    findUser (id: 1) {
+      birthDate
+      email
+      name
+      hash
+  }
+}`;
+
+export const userDatabase = {
```

Comment:
este userDatabase também não está com um nome muito claro, acredito que seria outro expected que poderia ficar la no it